server/routes/customers.js

```diff
@@ -62,4 +85,70 @@ router.post('/', (req, res) => {
   }
 });
 
+
+router.get('/:id/edit', (req, res) => {
+  const getId = parseInt(req.params.id);
+  // const userId = req.user.id;
+
+  // if (userId !== getId) {
+  //   return res.status(400).json({ status: Error, message: `Unable to edit ${getId}` });
+  // } else {
+    return new Customer({ id: getId })
+      .fetch({
+        require: true,
+        columns: ['first_name', 'last_name', 'username', 'email', 'state', 'city', 'zip_code']
+      })
+      .then(customer => {
+        const custObj = customer.serialize();
+        return res.json(custObj);
+      })
+      .catch(err => {
+        return res.status(500).json({ message: err.message, code: err.code });
+      });
+  // }
+});
+
+
+router.put('/:id/edit', (req, res) => {
+  const getId = parseInt(req.params.id);
+  const userId = req.user.id;
+  const { first_name, last_name, username, email, state, city, zip_code } = req.body;
+
+  if (userId !== getId) {
+    return res.status(400).json({ status: Error, message: `Unable to edit ${getId}` });
```

Comment:
usually you would return a `401` here which says someone is `Not Authorized` and no message. It's basically slamming the door shut with no feedback on someone doing something they're not authorized to do (to someone else's account)